docs/sources/developers/http_api/dashboard.md

```diff
@@ -20,18 +20,519 @@ title: Dashboard HTTP API
+| `dashboards:create` | <ul><li>`folders:*`</li><li>`folders:uid:*`</li></ul>                                                   |
+| `dashboards:write`  | <ul><li>`dashboards:*`</li><li>`dashboards:uid:*`</li><li>`folders:*`</li><li>`folders:uid:*`</li></ul> |
+{ .no-spacing-list }
+<!-- prettier-ignore-end -->
+
+
+**Example Create Request**:
+
+```http
+POST /apis/dashboard.grafana.app/v1/namespaces/default/dashboards HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+
+{
+  "metadata": {
+    "name": "gdxccn",
+    "annotations": {
+      "grafana.app/folder": "fef30w4jaxla8b"
+    },
+  },
+  "spec": {
+    "editable": true,
+    "schemaVersion": 41,
+    "title": "New dashboard",
+  }
+}
+```
+
+JSON Body schema:
+- **metadata.name** – The Grafana [unique identifier]({{< ref "#identifier-id-vs-unique-identifier-uid" >}}). If you do not want to provide this, set metadata.generateName to the prefix you would like for the uid.
+- **metadata.annotations.grafana.app/folder** - Optional field, the unique identifier of the folder under which the dashboard should be created.
+- **spec** – Details on what can be set in the spec can be found [above]({{< ref "#dashboard-schema" >}}).
+
+**Example Response**:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json; charset=UTF-8
+Content-Length: 485
+
+{
+  "kind": "Dashboard",
+  "apiVersion": "dashboard.grafana.app/v1",
+  "metadata": {
+    "name": "gdxccn",
+    "namespace": "default",
+    "uid": "965b82ca-c0fa-43f9-a87e-e10c4ee55023",
+    "resourceVersion": "1",
+    "generation": 1,
+    "creationTimestamp": "2025-03-07T02:48:18Z",
+    "annotations": {
+      "grafana.app/createdBy": "service-account:cef2t2rfm73lsb",
+      "grafana.app/folder": "fef30w4jaxla8b"
+    },
+    "managedFields": [
+      {
+        "manager": "curl",
+        "operation": "Update",
+        "apiVersion": "dashboard.grafana.app/v1alpha1",
+        "time": "2025-03-07T02:48:18Z",
+        "fieldsType": "FieldsV1",
+        "fieldsV1": {
+          "f:spec": {
+            "f:editable": {},
+            "f:schemaVersion": {},
+            "f:title": {}
+          }
+        }
+      }
+    ]
+  },
+  "spec": {
+    "editable": true,
+    "schemaVersion": 41,
+    "title": "New dashboard"
+  }
+}
+```
+
+Status Codes:
+
+- **201** – Created
+- **400** – Errors (invalid json, missing or invalid fields, etc)
+- **401** – Unauthorized
+- **403** – Access denied
+- **409** – Conflict (dashboard with the same uid already exists)
+
+## Update Dashboard
+
+`PUT /apis/dashboard.grafana.app/v1/namespaces/:namespace/dashboards/:uid`
+
+Updates an existing dashboard via the dashboard uid.
+
+- namespace: to read more about the namespace to use, see the [API overview]({{< ref "apis" >}}).
+- uid: the unique identifier of the dashboard to update. this will be the _name_ in the dashboard response
```

Comment:
@irenerl24 any ideas of how I could word this better? This is one the pieces that feels very confusing to me